storm-server/src/main/java/org/apache/storm/daemon/supervisor/BasicContainer.java

```diff
@@ -79,6 +79,7 @@
     protected final long lowMemoryThresholdMB;
     protected final long mediumMemoryThresholdMb;
     protected final long mediumMemoryGracePeriodMs;
+    private static int port = 5006;  // TODO: Roshan: remove this after stabilization
```

Comment:
Yes please make sure this is removed, as leaving it in place is a security vulnerability.